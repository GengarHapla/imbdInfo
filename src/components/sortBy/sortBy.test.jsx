import React from "react";
import SortBy from "./sortBy";

describe("SortBy component renders correctly", () => {
  it("when no props", () => {
    expect(<SortBy />).toMatchSnapshot();
  });

  it("when props with default option", () => {
    const props = {
        default: 'DATE'
    };

    expect(<SortBy {...props} />).toMatchSnapshot();
  });
});
