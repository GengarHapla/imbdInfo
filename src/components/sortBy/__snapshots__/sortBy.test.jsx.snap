// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SortBy component renders correctly when no props 1`] = `<SortBy />`;

exports[`SortBy component renders correctly when props with default option 1`] = `
<SortBy
  default="DATE"
/>
`;
